workspace.json

```diff
@@ -5,6 +5,10 @@
       "root": "apps/a11y-tests",
       "projectType": "application"
     },
+    "bundle-size": {
+      "root": "apps/bundle-size",
+      "projectType": "application"
```

Comment:
as previously mentioned -> IMO it should be a package -> thus projectType `library`

Reply:
Done in ca8f2a9